dolphinscheduler-common/src/main/resources/common.properties

```diff
@@ -18,7 +18,7 @@
 # user data local directory path, please make sure the directory exists and have read write permissions
 data.basedir.path=/tmp/dolphinscheduler
 
-# resource storage type: HDFS, S3, NONE
+# resource storage type: HDFS, HDFS_LOCAL, S3, NONE
```

Comment:
I think we should directly use `LOCAL` instead of `HDFS_LOCAL` here, It is more easier to understand